<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="leNombre">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leApellido">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leEdad">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAgregar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>520</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leClave">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lNombre">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="lApellido">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido</string>
    </property>
   </widget>
   <widget class="QLabel" name="lEdad">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>490</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Edad</string>
    </property>
   </widget>
   <widget class="QLabel" name="lClase">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>490</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Clave</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tDatos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>401</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Clave</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pbEliminar">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>520</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar Usuario</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
